src/ui/components/DashKit/plugins/Control/Control.tsx

```diff
@@ -972,9 +881,44 @@ class Control extends React.PureComponent<PluginControlProps, PluginControlState
         const preselectedContent = [{title: selectedValue, value: selectedValue}];
         // @ts-ignore
         const content = loadedData?.uiScheme?.controls[0].content;
+        const emptyPaceholder = Utils.isEnabledFeature(Feature.EmptySelector)
+            ? i18n('placeholder_empty')
+            : undefined;
 
         const preparedValue = unwrapFromArrayAndSkipOperation(this.actualParams[fieldId]);
 
+        // for first initialization of control
+        const initialValidationError = isValidationError({
```

Comment:
On init we already writing validation error and when then overwriting placeholder if feature is enabled. But i have a few questions.
Are we getting invalid field from the start even before user has interacted with field?
If my suggestion is true, than i think the better way is to use placeholder if we can't show otherwise required field and only after change we could pass validation error property.